### ISSUE TITLE
Double (s) and Float (s) have String constants for Nan, Infinity, Negative Infinity,

### ISSUE THREAD

**Issue by michaelcheers** [bridgedotnet/Bridge#1379]
### Expected

System.Double  (or maybe Bridge.Double considering #1293) (3 times)
System.Single (or maybe Bridge.Double considering #1293) (3 times)

### Actual

String (6 times)

### Steps To Reproduce
[Live Bridge](http://live.bridge.net/#f82a3f28b658e025d37e2b1574c1dcbb)
[Dot Net Fiddle](https://dotnetfiddle.net/Z8lVf2)
```csharp
/// <summary>
/// Creates a simple class
/// </summary>
public class App
{
    // Use the [Ready] attribute to automatically
    // run this method when the page is ready.
    [Ready]
    public static void Main()
    {
        object value = null;
        for (int time = 0; time < 6; time++)
        {
            switch (time)
            {
                case 0:
                {
                    value = 0.0 / 0.0;
                    break;
                }
                case 1:
                {
                    value = 1.0 / 0.0;
                    break;
                }
                case 2:
                {
                    value = -1.0 / 0.0;
                    break;
                }
                case 3:
                {
                    value = 0.0f / 0.0f;
                    break;
                }
                case 4:
                {
                    value = 1.0f / 0.0f;
                    break;
                }
                case 5:
                {
                    value = -1.0f / 0.0f;
                    break;
                }
            }
            Console.WriteLine(value.GetType().FullName);
        }
    }
}
```

**Comment by LeonidVeriga** [comment]
@michaelcheers, thank you for reporting this problem, very nice catch.

**Comment by LeonidVeriga** [comment]
We are not sure about fixing this to output `System.Double` in `1.14` but will try to fix this `object value = 0.0 / 0.0;` to be translated into `var value = NaN;` (instead of `var value = "NaN";`)

**Comment by LeonidVeriga** [comment]
In the current implementation, when getting type for consts:
```cs
0.0 / 0.0;
1.0 / 0.0;
1.0 / 0.0;
0.0f / 0.0f;
1.0f / 0.0f;
1.0f / 0.0f;
```
They are considered to be `System.Double`
